brave/browser-laptop
# Disables funding with debit/credit card. Only affects Coinbase

## Test plan
1. Be on an IP that is geo-located in the USA (via VPN or physically being in USA)
2. Go to Preferences > Payments
3. Click "Add funds"
4. "Fund with debit/credit" button should be disabled; appropriate messaging should be displayed 
<img width="669" alt="screen shot 2017-06-07 at 11 14 40 pm" src="https://user-images.githubusercontent.com/4733304/26914812-226b20f4-4bd8-11e7-9daf-86efc8b5cd07.png">

## Description
Disables funding with debit/credit card. Only affects Coinbase (ex: customers in USA)

Fixes https://github.com/brave/browser-laptop/issues/9327

Auditors: @cezaraugusto, @NejcZdovc 

Submitter Checklist:


Reviewer Checklist:

Tests
